client/asset/dcr/dcr.go

```diff
@@ -2444,7 +2444,7 @@ func (dcr *ExchangeWallet) sendRegFee(addr dcrutil.Address, regFee, netFeeRate u
 	}
 	coins, _, _, _, err := dcr.fund(enough)
 	if err != nil {
-		return nil, 0, fmt.Errorf("unable to pay registration fee of %s DCR with fee rate of %d atoms/byte: %w",
+		return nil, 0, fmt.Errorf("Insufficient funds for fee rate. Unable to pay registration fee of %s DCR with fee rate of %d atoms/byte: %w",
```

Comment:
I think must be lowercase `i` in order to match the string.

But really, would be good to have an error code or something, as matching strings is very fragile.

Also, one day fees may be payable by different coins. So, other assets must also pay attention to their error messages. Also, this may cause problems with internationalization.